Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

aaronbell opened the pull request:
216d8b0: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/bf8a7c78d833e4aa6e70f8d162526e5fd0b6aac9.

 199373e: [gftools-packager] ofl/shipporiantique remove METADATA "source".  google/fonts#2587

 7d0b4f8: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/bf8a7c78d833e4aa6e70f8d162526e5fd0b6aac9.

 e0503bb: [gftools-packager] ofl/shipporiantiqueb1 remove METADATA "source".  google/fonts#2587

m4rc1e reviewed (changes requested):
Is there a reason why the nbspace and space have different widths? for latin fonts we set them so they are both the same.

aaronbell commented:
No, they should be the same. I thought I had correctly set them. Will investigate.

m4rc1e commented:
Thanks and keep em coming.

aaronbell commented:
Updated

## Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

---
 e49f72d: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/bf8a7c78d833e4aa6e70f8d162526e5fd0b6aac9.

 64b54b7: [gftools-packager] ofl/shipporiantique remove METADATA "source".  google/fonts#2587

 c7ee06c: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/bf8a7c78d833e4aa6e70f8d162526e5fd0b6aac9.

 e7d7877: [gftools-packager] ofl/shipporiantiqueb1 remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

---
 17cef8b: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/c513fcbb513c8443af440be6dd3effe33e434719.

 44aee41: [gftools-packager] ofl/shipporiantique remove METADATA "source".  google/fonts#2587

 997d160: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/c513fcbb513c8443af440be6dd3effe33e434719.

 9465de6: [gftools-packager] ofl/shipporiantiqueb1 remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

---
 0d1034c: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/89859ee09d2de705a6285583068e1af699433205.

 5e7a27b: [gftools-packager] ofl/shipporiantique remove METADATA "source".  google/fonts#2587

 badfbb6: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/89859ee09d2de705a6285583068e1af699433205.

 e8473ba: [gftools-packager] ofl/shipporiantiqueb1 remove METADATA "source".  google/fonts#2587

aaronbell commented:
A couple of last changes were made by the designer.

aaronbell commented:
Updated

## Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

---
 56ce53c: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/d2774b4ab2a408000706adbeea02c23ee8f52b7d.

 b84759c: [gftools-packager] ofl/shipporiantique remove METADATA "source".  google/fonts#2587

 af76232: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/d2774b4ab2a408000706adbeea02c23ee8f52b7d.

 5e2d36e: [gftools-packager] ofl/shipporiantiqueb1 remove METADATA "source".  google/fonts#2587

aaronbell commented:
Updated

## Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

---
 ca0af79: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/3510d198da0c8ec766558e06c7d30bd7ec2b7645.

 64e9f4d: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/3510d198da0c8ec766558e06c7d30bd7ec2b7645.

aaronbell commented:
Updated

## Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added; Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

---
 41c6926: [gftools-packager] Shippori Antique: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/1f4014d717cd8ce33733858836ea4337bbf0280e.

 d3eb523: [gftools-packager] Shippori Antique B1: Version 2.000; ttfautohint (v1.8.3) added

* Shippori Antique B1 Version 2.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontdasu/ShipporiAntique.git at commit https://github.com/fontdasu/ShipporiAntique/commit/1f4014d717cd8ce33733858836ea4337bbf0280e.

aaronbell commented:
Updating font files per request from foundry

RosaWagner commented:
Files size is super large
